qiita_pet/templates/study_base.html

```diff
@@ -15,12 +16,26 @@
 
   function fill_artifact(aid) {
     $("#study-main").html("ARTIFACT " + aid + " HTML STUFF");
+    $.get("/admin/artifact/", {artifact_id: +aid })
```

Comment:
That's incorrect, you're actually also casting in the python code: https://github.com/biocore/qiita/pull/1593/files#diff-ea0af58e24e01961bafc83e4c0124a05R43